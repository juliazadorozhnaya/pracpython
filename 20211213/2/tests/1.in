[95, 90, 78, 74, 65, 64, 60, 55, 40, 39, 36, 31, 24, 15, 8, 1]
