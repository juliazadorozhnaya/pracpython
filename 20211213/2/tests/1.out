[1, 8, 15, 24, 31, 36, 39, 40, 55, 60, 64, 65, 74, 78, 90, 95]
